CondCore/JetMETPlugins/plugins/JetResolution_PayloadInspector.cc

```diff
@@ -97,9 +120,31 @@ namespace JME {
   public:
     JetResolutionVsPt()
         : cond::payloadInspector::Histogram1D<JetResolutionObject, SINGLE_IOV>(
-              "Jet Energy Resolution", "p_T", NBIN_PT, MIN_PT, MAX_PT, "Resolution") {}
+              "Jet Energy Resolution", "p_T", NBIN_PT, MIN_PT, MAX_PT, "Resolution") {
+      cond::payloadInspector::PlotBase::addInputParam("Jet_Pt");
+      cond::payloadInspector::PlotBase::addInputParam("Jet_Eta");
+      cond::payloadInspector::PlotBase::addInputParam("Jet_Rho");
+    }
 
     bool fill() override {
+      double par_Pt = 100.;
+      double par_Eta = 1.;
+      double par_Rho = 20.;
+
+      auto paramValues = cond::payloadInspector::PlotBase::inputParamValues();
+      auto ip = paramValues.find("Jet_Pt");
+      if (ip != paramValues.end()) {
+        par_Pt = std::stod(ip->second);
```

Comment:
This `par_Pt` is never used (overwritten at [L166](https://github.com/cms-sw/cmssw/pull/41529/files#diff-2f814a0d8ef5675be3b3246f3f0b746b66a336a096f3c2177e051af3193dfe29R166)): please check

Reply:
same as L96, removing the unused inputs pushed.